src/karma.config.js

```diff
@@ -91,7 +91,10 @@ module.exports = function(config) {
 
       // CSS bundles, relied upon by accessibility tests (eg. for color-contrast
       // checks).
-      { pattern: '../build/styles/{annotator,sidebar}.css', watched: false },
+      {
+        pattern: '../build/styles/{annotator,icomoon,sidebar}.css',
```

Comment:
When the `createShadowRoot` utility loads styles into the shadow root, it copies the stylesheet URLs from the host page. In order for this to work, all of the style bundles used ("annotator.css" and "icomoon.css") need to be loaded. Previously icomoon was not loaded in the tests as it was never used by any of them.

Once we are using `SvgIcon` everywhere, this can go away.